src/main/java/edu/cornell/kfs/coa/document/CuSubAccountMaintainableImpl.java

```diff
@@ -358,10 +358,10 @@ private boolean isIcrSectionDataChanged(List<A21IndirectCostRecoveryAccount> new
     /**
      * After a copy is done set specific fields on {@link SubAccount} to default values
      *
-     * @see org.kuali.rice.kns.maintenance.KualiMaintainableImpl#processAfterCopy(MaintenanceDocument, Map)
+     * @see org.kuali.rice.kns.maintenance.KualiMaintainableImpl#processAfterCopy(org.kuali.rice.kns.document.MaintenanceDocument, Map)
      */
     @Override
-    public void processAfterCopy(org.kuali.rice.kns.document.MaintenanceDocument  document, Map<String, String[]> parameters) {
+    public void processAfterCopy(MaintenanceDocument  document, Map<String, String[]> parameters) {
```

Comment:
org.kuali.kfs.kns.document.MaintenanceDocument extends org.kuali.kfs.krad.document.MaintenanceDocument, so I think this is ok, at least for now; if we find we need to change something to get the application to start/work, we can do it then I think. Thanks.